bundles/org.openhab.automation.pythonscripting/README.md

```diff
@@ -0,0 +1,495 @@
+# Python 3 Binding
+
+This add-on provides support for Python 3 that can be used as a scripting language within automation rules.
+It is based on [GraalPy](https://www.graalvm.org/python/) from the [GraalVM project](https://www.graalvm.org/).
+
+Also included is [openhab-python](https://github.com/HolgerHees/openhab-python), a python library to support automation in openHAB. It provides convenient access to common core openHAB functions that make the full range of Java APIs easily accessible and usable
+
+[[toc]]
+
+## Creating Python Scripts
+
+When this add-on is installed, you can select Python3 as a scripting language when creating a script action within the rule editor of the UI.
+
+Alternatively, you can create scripts in the `automation/python` configuration directory. If you create an empty file called `test.py`, you will see a log line with information similar to:
+
+```text
+... [INFO ] [ort.loader.AbstractScriptFileWatcher]] - (Re-)Loading script '/openhab/conf/automation/python/test.py'
+```
+
+To enable debug logging, use the [console logging]({{base}}/administration/logging.html) commands to
+enable debug logging for the automation functionality:
+
+```text
+log:set DEBUG org.openhab.automation.pythonscripting
+```
+
+## Scripting Basics
+
+Lets start with a simple script
+
+```python
+from openhab import rule
+from openhab.triggers import GenericCronTrigger
+
+@rule( triggers = [ GenericCronTrigger("*/5 * * * * ?") ] )
+class Test:
+    def execute(self, module, input):
+        self.logger.info("Rule was triggered")
+```
+
+or another one, using the [scope module](#module-scope)
+
+```python
+from openhab import rule
+from openhab.triggers import ItemCommandTrigger
+
+import scope
+
+@rule( triggers = [ ItemCommandTrigger("Item1", scope.ON) ] )
+class Test:
+    def execute(self, module, input):
+        self.logger.info("Rule was triggered")
+```
+
+::: tip Note
+By default, the scope, Registry and logger is automatically imported for UI based rules
+:::
+ 
+
+## `PY3` Transformation
```

Comment:
Once this is merged, we can update the UI provide doc links, code snippets etc, see https://github.com/openhab/openhab-webui/blob/main/bundles/org.openhab.ui/web/src/assets/transformations.js.

Reply:
@HolgerHees Do you want to do this yourself?